velox/tpch/gen/TpchGen.h

```diff
@@ -80,11 +80,10 @@ TypePtr resolveTpchColumn(Table table, const std::string& columnName);
 ///  o_comment: VARCHAR
 ///
 RowVectorPtr genTpchOrders(
+    memory::MemoryPool* pool,
     size_t maxRows = 10000,
     size_t offset = 0,
-    double scaleFactor = 1,
-    memory::MemoryPool* pool =
```

Comment:
Any particular reason to move 'pool' parameter to the front? It seems just removing the default value would be sufficient.

Reply:
We can't because all the preceding input parameters have defaults :)